integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -206,28 +206,28 @@ def test_multiplication_ansi_overflow_for_decimal():
         exception_str)
 
 _gen_gen_lit_tuple_list_overflow_for_multiply = [
-    (ByteGen(min_val=30, max_val=50, special_cases=[]), ByteGen(min_val=50, max_val=127, special_cases=[]), -50),  # 30 * 50 > 127 (Byte.MaxValue)
-    (ShortGen(min_val=-32768, max_val=-30000, special_cases=[]), ShortGen(min_val=2, max_val=32767, special_cases=[]), 2), # 30000 * 2 > 32767 (Short.MaxValue)
-    (IntegerGen(min_val=2147483647//2, max_val=2147483647, special_cases=[]), IntegerGen(min_val=3, max_val=2147483647, special_cases=[]), 3) , # 2147483647//2 * 3 > 2147483647 (Int.MaxValue)
-    (LongGen(min_val=9223372036854775807//2, max_val=9223372036854775807, special_cases=[]), LongGen(min_val=3, max_val=9223372036854775807, special_cases=[]), 2147483647)] # 9223372036854775807//2 * 3 > 9223372036854775807(Long.MaxValue)
+    (ByteGen(nullable=False, min_val=30, max_val=50, special_cases=[]), ByteGen(nullable=False, min_val=50, max_val=127, special_cases=[]), -50),  # 30 * 50 > 127 (BYTE_MAX)
+    (ShortGen(nullable=False, min_val=SHORT_MIN, max_val=-30000, special_cases=[]), ShortGen(nullable=False, min_val=2, max_val=SHORT_MAX, special_cases=[]), 2), # 30000 * 2 > 32767 (SHORT_MAX)
+    (IntegerGen(nullable=False, min_val=INT_MIN, max_val=INT_MIN//2, special_cases=[]), IntegerGen(nullable=False, min_val=3, max_val=INT_MAX, special_cases=[]), 3) , # INT_MIN//2 * 3 < INT_MIN
```

Comment:
Why is nullable not allowed ?

Reply:
If left operand and right operand are both null, then CPU/GPU will not throw exception because result is null.